plugins/source-verifier/api.ts

```diff
@@ -1,5 +1,5 @@
 export interface Network {
-  name: 'ropsten' | 'mainnet' | 'rinkeby' | 'goerli'
+  name: 'ropsten' | 'mainnet' | 'rinkeby' | 'goerli' | 'kovan'
```

Comment:
I thought we change it so the second params is either a string (network name) or an id (network id).

Reply:
Network object is this, yeah, then this is not needed.